src/tools/compiletest/src/header.rs

```diff
@@ -581,6 +581,16 @@ impl TestProps {
             self.incremental = true;
         }
 
+        if config.mode == Mode::Crashes {
+            // we don't want to pollute anything with backtrace-files
+            // also turn off backtraces in order to save some execution
+            // time on the tests; we only need to know IF it crashes
+            self.rustc_env = vec![
+                ("RUST_BACKTRACE".to_string(), "0".to_string()),
+                ("RUSTC_ICE".to_string(), "0".to_string()),
+            ];
```

Comment:
This will unset any other possible envs and left only this two, can it be wrong?

Reply:
I don't see why I would have other env vars influencing test outcome :thinking:

Reply:
Looks like it possible to set env vars via "rustc-env" directive, but looks like this will unset it?